src/components/blocksmith-field/components/poll-block-modal/PollBlockPreview.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Loading } from 'components';
+import useNode from 'plugins/ncr/components/useNode';
+import './preview.scss';
+
+
+const PollBlockPreview = ({ className = '', nodeRef }) => {
+  const { node, pbjxError } = useNode(nodeRef, false);
```

Comment:
looks like nodeRef can potentially be the string 'null' from the poll-block-modal, in useNode I see

`  useEffect(() => {
    if (!nodeRef || consistent || !cachedNode) {
      return noop;
    }

    setNode(cachedNode.freeze());
    setError(null);
    setStatus(constants.STATUS_FULFILLED);
  }, [nodeRef, cachedNode, consistent]);`

maybe check for the string 'null' too and return early

Reply:
@bhsiao55 the poll block modal has a 2 step screen. the first screen you must select the poll and the second screen shows the preview. that being said, you will always have a nodeRef in order to get to the preview.